specification/web/resource-manager/readme.md

```diff
@@ -478,8 +478,8 @@ input-file:
 - Microsoft.Web/stable/2018-02-01/CommonDefinitions.json
 - Microsoft.Web/stable/2018-02-01/WebApps.json
 - Microsoft.Web/stable/2018-02-01/ResourceProvider.json
-- Microsoft.Web/stable/2016-09-01/AppServicePlans.json
-- Microsoft.Web/stable/2016-03-01/Provider.json
-- Microsoft.Web/stable/2016-03-01/ResourceProvider.json
-- Microsoft.Web/stable/2016-03-01/Recommendations.json
+- Microsoft.Web/stable/2018-02-01/AppServicePlans.json
+- Microsoft.Web/stable/2018-02-01/Provider.json
+- Microsoft.Web/stable/2018-02-01/ResourceProvider.json
+- Microsoft.Web/stable/2018-02-01/Recommendations.json
```

Comment:
@bganapa - Will the above changes work for Azure Stack?

Reply:
looks good to me as webapps team says 2018-02-01 is good for all the resource types